dts/arm/nxp/nxp_s32k344_m7.dtsi

```diff
@@ -828,6 +828,20 @@
 			};
 		};
 
+		flexio0: flexio@40324000 {
+			compatible = "nxp,flexio";
+			reg = <0x40324000 0x4000>;
+			interrupts = <139 0>;
+			clocks = <&clock NXP_S32_FLEXIO0_CLK>;
+			status = "disabled";
+
+			pwm {
```

Comment:
To be consistent with label names used for spi, please change this to flexio0_pwm